routes/blog/[slug].tsx

```diff
@@ -20,9 +20,10 @@ export const handler: Handlers<BlogPostPageData, State> = {
 
 export default function PostPage(props: PageProps<BlogPostPageData>) {
   const { post } = props.data;
-  const date = new Date(post.publishedAt).toLocaleDateString("en-US", {
-    dateStyle: "long",
-  });
+  const date = post.publishedAt.toString() !== "Invalid Date" &&
```

Comment:
whenever we had an `Invalid Date`, it would get rendered as well.  
this aligns our logic with `/blog/index.tsx`, where we explicitly check for `Invalid Date`.